### PR TITLE
Added Bank Transfer as Payment Icon

### PR DESCRIPTION
**Pull request by fthobe**
#### What?
added common european and chinese payment method of bank transfer to the icons, schema and lang file.

### CODE REVIEW

File: schemaTranslations.json [hunk 1]
```diff
@@ -811,11 +811,11 @@
...
-    "default": "Show Visa",
-    "fr": "Afficher Visa",
-    "it": "Mostra Visa",
-    "uk": "Показати Visa",
-    "zh": "显示 Visa"
+    "default": "Show Wire Transfer",
+    "fr": "Afficher virement bancaire",
+    "it": "Mostra Bonifico",
+    "uk": "Показати банківський переказ",
+    "zh": "显示 电汇"
```
Comment (BC-tymurbiedukhin) [inline schemaTranslations.json:818]: Do we have correct field name and translation values here?

**Review by BC-tymurbiedukhin — COMMENTED** [review]
Could you please add screenshots how it looks like for mobile, tablet and desktop versions?

File: schemaTranslations.json [hunk 2]
```diff
@@ -831,6 +831,13 @@
     "uk": "Показати Google Pay",
     "zh": "显示 Google Pay"
   },
+  "i18n.ShowWireTransfer": {
+    "default": "Show Wire Transfer",
+    "fr": "Afficher Wire Transfer",
+    "it": "Mostra Wire Transfer",
+    "uk": "Показати Wire Transfer",
+    "zh": "显示 Wire Transfer"
+  },
```
Comment (BC-tymurbiedukhin) [inline schemaTranslations.json:840]: Do we need to have unused field here?

File: schemaTranslations.json (hunk 1 above)
Comment (fthobe) [reply schemaTranslations.json]: @BC-tymurbiedukhin Ahhh... think I messed up here, I will fix it and make a new pull request.

**Review by fthobe — COMMENTED** [review]
Well, if someone wants them he can grab the strings and is good to go!

**Comment by fthobe** [comment]
Hey, I fixed the schemaTranslation.json in my repo. 
@BC-tymurbiedukhin regarding screenshots: they roll exactly like the others as I expressively matched proportion / ratios of the svg to the existing svgs.

**Pull request closed without merging**